src/main/java/com/example/bookstore/model/Book.java

```diff
@@ -0,0 +1,22 @@
+package com.example.bookstore.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
+public class Book {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
```

Comment:
you must add constraint "not null" for title, author, isbn, price. And "unique" for isbn